src/elements/flip-card/flip-card-summary/flip-card-summary.scss

```diff
@@ -0,0 +1,80 @@
+@use '../../core/styles' as sbb;
+
+// Box-sizing rules contained in typography are not traversing Shadow DOM boundaries. We need to include box-sizing mixin in every component.
+@include sbb.box-sizing;
+
+:host {
+  --sbb-flip-card-summary-opacity: 1;
+  --sbb-flip-card-summary-pointer-events: all;
+
+  display: contents;
+}
+
+.sbb-flip-card-summary--wrapper {
+  display: grid;
+  position: absolute;
+  pointer-events: var(--sbb-flip-card-summary-pointer-events);
+  opacity: var(--sbb-flip-card-summary-opacity);
+  border-radius: var(--sbb-flip-card-border-radius);
+  overflow: hidden;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto 1fr;
+
+  :host([image-alignment='after']) & {
+    @include sbb.mq($from: small) {
+      grid-template-columns: repeat(3, 1fr);
+      grid-template-rows: 1fr;
+    }
+
+    @include sbb.mq($from: medium) {
+      grid-template-columns: repeat(2, 1fr);
+      grid-template-rows: 1fr;
+    }
+  }
+
+  width: 100%;
```

Comment:
Nitpick; With the upcoming scss change, rules should not be separated by nested blocks, unless absolutely necessary.